common/djangoapps/microsite_configuration/backends/filebased.py

```diff
@@ -0,0 +1,199 @@
+"""
```

Comment:
I think lets keep it as it is. Django also name many backends as `filebased` i.e. this cache backend
https://docs.djangoproject.com/en/1.9/topics/cache/